server/graphql/common/expenses.ts

```diff
@@ -617,6 +619,28 @@ export const canMarkAsIncomplete: ExpensePermissionEvaluator = async (
   }
 };
 
+/**
+ * Returns true if user is allowed to change the accounting category of the expense
+ */
+export const canEditExpenseAccountingCategory = async (
+  req: express.Request,
+  expense: Expense,
+  options = { throw: false },
+): Promise<boolean> => {
+  return remoteUserMeetsOneCondition(
+    req,
+    expense,
+    [
+      // Host admins and accountants can always change the accounting category.
+      isHostAdmin,
+      isHostAccountant,
+      // Otherwise, we fallback on the default edit permissions (must be a collective/fromCollective admin)
+      canEditExpense,
```

Comment:
This will enable `CollectiveAdmin to edit paid expenses, not sure if we want this.

Reply:
If the expense is STATUS=PAID, `canEditExpense` will let collective admins pass only if the expense is a charge - aka when they upload the documentation for the virtual card - which seems like the perfect moment to set the accounting category.